client/state/data-layer/wpcom/read/lists/index.js

```diff
@@ -93,7 +95,9 @@ registerHandlers( 'state/data-layer/wpcom/read/lists/index.js', {
 			},
 			onSuccess: ( action, response ) => [
 				receiveUpdatedListDetails( response ),
-				successNotice( translate( 'List updated successfully!' ) ),
+				successNotice( translate( 'List updated successfully.' ), {
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.